src/Middleware/OutputCaching/src/Policies/SetCacheKeyPrefixPolicy.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.OutputCaching;
+
+/// <summary>
+/// A policy that sets the cache key prefix using the specified value.
+/// </summary>
+internal sealed class SetCacheKeyPrefixPolicy : IOutputCachePolicy
+{
+    private readonly Func<HttpContext, CacheVaryByRules, CancellationToken, ValueTask>? _varyByAsync;
```

Comment:
Is this actually nullable? Looks like it never will be so you can get rid of the null check and nullable annotation